blank-canvas-blocks/experimental-theme.json

```diff
@@ -97,20 +101,19 @@
 					"horizontal": "20px",
 					"vertical": "20px"
 				},
-				"padding": {
-					"horizontal": "20px",
-					"vertical": "20px"
-				},
-				"width": {
-					"default": "620px",
-					"wide": "1000px"
-				},
 				"button": {
-					"fontWeight": "normal",
-					"fontFamily": "var(--wp--preset--font-family--base)",
-					"fontSize": "var(--wp--preset--font-size--small)",
-					"borderRadius": "4px",
+					"typography": {
+						"fontWeight": "normal",
+						"fontFamily": "var(--wp--preset--font-family--base)",
+						"fontSize": "var(--wp--preset--font-size--small)",
+						"lineHeight": ""
```

Comment:
Should this value be empty?

Reply:
Probably not, and likely also needs to be referenced below in the "styles" block.

Reply:
These have been returned.